src/UserManager.ts

```diff
@@ -598,8 +601,7 @@ export class UserManager {
             logger.debug("storing user");
             const storageString = user.toStorageString();
             await this.settings.userStore.set(this._userStoreKey, storageString);
-        }
-        else {
+        } else {
```

Comment:
please avoid changing code style,

Reply:
I am sorry, this is something that my IDE does if not otherwise configured. Perhaps you should consider enforcing this specific rule as well via the .editorconfig file if possible.